Elastiflix/docker-compose-elastic-otel-demo.yml

```diff
@@ -130,8 +124,7 @@ services:
     ports:
       - "3001:3001"
     environment:
-      # - API_ENDPOINT_FAVORITES=favorite-python-elastic-manual:5000,favorite-go-elastic-manual:5000,favorite-java-elastic-auto:5000,favorite-java-elastic-manual:5000
-      - API_ENDPOINT_FAVORITES=favorite-go-elastic-manual:5000,favorite-java-elastic-auto:5000,favorite-java-elastic-manual:5000
+      - API_ENDPOINT_FAVORITES=favorite-java-elastic:5000,favorite-java-otel:5000,favorite-java-elastic-otel:5000
```

Comment:
any particular reason you're removing avoid requests to the `favorite-go-elastic-manual` service?

if required we could remove the service if not used

Reply:
I've removed the service as it's not relevant for the demo, keeping only java and nodes in the map (+the dotnet as it's the one implementing login).